### DISCUSSION TITLE
How transform word to uppercase?

### DISCUSSION THREAD

**Discussion by zer09** [L3MON4D3/LuaSnip#351] [Q&A] [answered]
How I can make the completed words to be all in uppercase?
I have something like this
```json
  "sample": {
    "prefix": "sample",
    "body": [
      "${1:param}",
      "'${1/(.*)/${1:/upcase}/}'"
    ],
    "description": "Sample demo"
  },
```
But the second line on the body is not transforming to uppercase

**Comment by L3MON4D3** [comment, marked as answer]
That's not possible yet via parsed snippets :/

You can get this nonetheless by creating the snippet in lua:
```lua
s("sample", fmt[[
	{}
	'{}'
]], {
	i(1, "param"), l(l._1:upper())
})
```

**Reply by zer09** [reply]
Got it, thanks!
